### PR TITLE
Add editable and selectable to ossBulkRegRows

### PR DESCRIPTION
**Pull request by Gseungmin**
edit and multiselct are added to ossBulkRegRows
Signed-off-by: Ji Seungmin <cswcsm02@gmail.com>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by riyenas0925** [comment]
Please change all comments to English

File: src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp [hunk 1]
```diff
@@ -36,11 +48,36 @@
...
+            //리스트 데이터 검사
+            listDataForValid.forEach((listData) => {
+                ossData[index][listData] = row[listData].trim()
+                if (ossData[index][listData] == "") {
+                    ossData[index][listData] = []
+                } else {
+                    ossData[index][listData] = ossData[index][listData].split(",")
+                }
+            })
+        })}
```
Comment (FRESH-TUNA) [inline src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp:69]: Indentation of 69 line doesn't look good.
It should be matched with `function dataValidCheck(mainData) {`

File: src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp [hunk 2]
```diff
@@ -114,8 +163,35 @@
...
+                    editable()
+
+                    $("#list").jqGrid('editRow', rowid);
+                    _mainLastsel = rowid;
+
+                    var nextCol = $("#list").jqGrid('getGridParam', 'colModel')[iCol].name
+                    var nextRow = rowid
+                    $('#' + nextRow + "_" + nextCol).focus();
+            }
+        }});
```
Comment (FRESH-TUNA) [inline src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp:194]: How about change indentation of code likes below?
```javascript
    ondblClickRow: function(rowid,iRow,iCol,e) {
        if(rowid) {
            if (rowid != _mainLastsel) {
                $("#list").jqGrid('saveRow', _mainLastsel);
            }
            editable()
            $("#list").jqGrid('editRow', rowid);
            _mainLastsel = rowid;
            var nextCol = $("#list").jqGrid('getGridParam', 'colModel')[iCol].name
            var nextRow = rowid
            $('#' + nextRow + "_" + nextCol).focus();
        }
    }
});
```

File: src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp (hunk 1 above)
Comment (Gseungmin) [reply src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp]: thank you for your review! I fixed commit right before

File: src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp (hunk 2 above)
Comment (Gseungmin) [reply src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp]: thank you for your review! I fixed commit right before

**Pull request merged**